tools/rosbag/CMakeLists.txt

```diff
@@ -8,19 +8,20 @@ endif()
 find_package(catkin REQUIRED COMPONENTS rosbag_storage rosconsole roscpp std_srvs topic_tools xmlrpcpp)
 find_package(Boost REQUIRED COMPONENTS date_time regex program_options filesystem)
 find_package(BZip2 REQUIRED)
+find_package(yaml-cpp REQUIRED)
```

Comment:
A proper dependency needs to be declared in the `package.xml` in order to correctly build this package and pass CI.